Description: Fix FTBFS on big-endian 64-bit architectures
 A long* pointer was incorrectly cast as an int*.
Origin: upstream, https://github.com/OpenMathLib/OpenBLAS/commit/b1d722fc0cf563298de8c5ae66fc04a1901d4bf1
Bug: https://github.com/OpenMathLib/OpenBLAS/issues/4633
Reviewed-by: Sébastien Villemot <sebastien@debian.org>
Last-Update: 2024-04-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/utest/test_extensions/xerbla.c
+++ b/utest/test_extensions/xerbla.c
@@ -39,7 +39,7 @@ static char *rout;
 
 static void F77_xerbla(char *srname, void *vinfo)
 {
-   int info=*(int*)vinfo;
+   blasint info=*(blasint*)vinfo;
 
    if (link_xerbla)
    {
@@ -85,4 +85,4 @@ void set_xerbla(char* current_rout, int
    lerr = TRUE;
    _info = expected_info;
    rout = current_rout;
-}
\ No newline at end of file
+}
